compiler/src/typed/oprint.re

```diff
@@ -738,8 +766,8 @@ and print_out_type_decl = (kwd, ppf, td) => {
     | Otyp_sum(constrs) =>
       fprintf(
         ppf,
-        " =@;<1 2>%a",
-        print_list(print_out_constr, ppf => fprintf(ppf, "@ | ")),
+        " {\n@;<2 2>%a\n}",
+        print_list(print_out_constr, ppf => fprintf(ppf, "\n  ")),
```

Comment:
And then theoretically this could be
```suggestion
        print_list(print_out_constr, ppf => fprintf(ppf, "@;  ")),
```

and it should always split because of the vertical box. These boxes are weird. Let me know if it works.